runtime/onert/backend/cpu/ops/DepthwiseConvolutionLayer.cc

```diff
@@ -84,11 +84,11 @@ void DepthwiseConvolutionLayer::convQuant8()
     getBuffer<uint8_t>(_output), _external_context->ruy_context());
 }
 
-void DepthwiseConvolutionLayer::convQuant8PerChannel()
+void DepthwiseConvolutionLayer::convQ8i()
```

Comment:
PerChannel sounds proper for me. What does suffix `i` mean here ?

Reply:
1) I would like to distinguish `uint8` and `int8`. `i` means `int8`. `u` means `uint8`.
2) `convQuant8PerChannel` is `int8` kernel which can handle both per-tensor and per-channel. Meanwhile, `uint8` kernel uses two different kernels for per-tensor and per-channel respectively.